app/workers/activitypub/followers_synchronization_worker.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class ActivityPub::FollowersSynchronizationWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: 'push', lock: :until_executed
+
+  def perform(account_id, url)
+    @account = Account.find_by(id: account_id)
+    return true if @account.nil?
+
+    ActivityPub::SynchronizeFollowersService.new.call(@account, url)
```

Comment:
Could be compacted to one line by doing `Account.find` and rescuing the error like we do in all other workers.

Reply:
This is a pattern I do not like that much as it could silently hide unexpected `ActiveRecord::RecordNotFound` exceptions.